SeleniumTasks/TestAmazonHomework/src/test/java/AmazonTest.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class AmazonTest extends BaseTest {
+    String amazon = "https://www.amazon.com/";
```

Comment:
Page URL should be written in uppercase letters and should clearly be indicated that it is a URL, otherwise what does it mean "amazon"?